### PR TITLE
Revert untranslated keys added to locales

### PR DESCRIPTION
**Pull request by HeavenVolkoff**

### CODE REVIEW

File: interface/locales/it/common.json [hunk 1]
```diff
@@ -587,9 +582,7 @@
   "system": "Sistema",
   "tag": "Tag",
   "tag_one": "Tag",
-  "tag_many": "Tags",
   "tag_other": "Tags",
-  "tags": "Tags",
```
Comment (HeavenVolkoff) [inline interface/locales/it/common.json:592]: Should we include these @utkubakir? They weren't originally in this translation file, but they have the same values as the others already included...

Comment (HeavenVolkoff) [reply interface/locales/it/common.json:592]: Or should we remove the other non translated keys as well

Comment (utkubakir) [reply interface/locales/it/common.json:592]: yeah this is a bit weird, If I remember correctly, you only need tag_one and tag_other if the count value is not changing the pluralization. I say remove it it's a duplicate.

https://www.i18next.com/translation-function/plurals

**Comment by artsiom-voitas** [comment]
Please, check General channel in Discord, before merging! 
"tag_" keys should stay

**Comment by HeavenVolkoff** [comment]
> Please, check General channel in Discord, before merging! "tag_" keys should stay

Ok, will restore the tag keys, thanks for the warning @artsiom-voitas

**Review by ameer2468 — APPROVED** [review]

**Pull request merged**